libevmasm/AssemblyItem.cpp

```diff
@@ -243,6 +243,18 @@ string AssemblyItem::getJumpTypeAsString() const
 	}
 }
 
+void AssemblyItem::setJumpType(string const& _jumpType)
+{
+	if (_jumpType == "[in]")
+		m_jumpType = JumpType::IntoFunction;
+	else if (_jumpType == "[out]")
+		m_jumpType = JumpType::OutOfFunction;
+	else if (_jumpType.empty())
+		m_jumpType = JumpType::Ordinary;
+	else
+		solThrow(AssemblyException, "Invalid jump type.");
```

Comment:
I think this is a part of the import functionality so I'd use the import exception here.
```suggestion
		solThrow(AssemblyImportException, "Invalid jump type.");
```